anyway/models.py

```diff
@@ -894,25 +894,36 @@ def set_critical(
         from anyway.widgets.all_locations_widgets.injured_count_by_severity_widget import (
             InjuredCountBySeverityWidget,
         )
-        from anyway.request_params import get_latest_accident_date
+        from anyway.request_params import get_latest_accident_date, LocationInfo
```

Comment:
models.py is a very general file. it specifies the database tables structure. I think it should not depend, or hold specific logic that uses these tables. Code that is related to Widgets should not be here, probably it should be somewhere under widgets.

Reply:
Thanks!
I agree, and I believe this change will better be in a separate pull request since this logic specific was there before.
@atalyaalon

Reply:
I agree, @EliorGigi you can create a separate issue and pr for this